### PR TITLE
New Package:py-responses

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-responses/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class PyResponses(PythonPackage):
+    """A utility library for mocking out the `requests` Python library."""
+
+    homepage = "https://github.com/getsentry/responses"
+    pypi     = "responses/responses-0.13.3.tar.gz"
+
+    maintainers = ['dorton21']
+
+    version('0.13.3', sha256='18a5b88eb24143adbf2b4100f328a2f5bfa72fbdacf12d97d41f07c26c45553d')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-responses/package.py:18]:
```python
depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-responses/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    depends_on('py-setuptools', type='build')
+    depends_on('py-mock', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-responses/package.py]:
```suggestion
    depends_on('py-mock', when='^python@:3.2.999', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-responses/package.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    """A utility library for mocking out the `requests` Python library."""
+
+    homepage = "https://github.com/getsentry/responses"
+    pypi     = "responses/responses-0.13.3.tar.gz"
+
+    maintainers = ['dorton21']
+
+    version('0.13.3', sha256='18a5b88eb24143adbf2b4100f328a2f5bfa72fbdacf12d97d41f07c26c45553d')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-responses/package.py:20]: Missing dep:
```python
depends_on('py-cookies', when='^python@:3.3.999', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Comment by adamjstewart** [comment]
Tests are failing because the `py-cookies` package does not yet exist.

**Comment by dorton21** [comment]
> Tests are failing because the `py-cookies` package does not yet exist.

My bad. This is the related PR for py-cookies: #24084

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**